### ISSUE TITLE
special/webpack: babel-loader presets are not detected

### ISSUE THREAD

**Issue by sveyret** [depcheck/depcheck#448]
### Bug Description
When a babel preset is used in webpack configuration file, it is not detected as a dependency.

### Code snippets
Where the dependency is used:

```
module.export = {
    module: {
      rules: [
        {
          test: /\.js$/, use: {
            loader: 'babel-loader',
            options: { presets: [
              ['@babel/preset1', { option: 'option' }],
              '@babel/preset2'
            ]},
          }
        },
      ],
    },
}
```

Where the dependency is listed in package.json:

```
"devDependencies": {
  "@babel/preset1": "7.6.3",
  "@babel/preset2": "7.6.3",
}
```

### Versions
 - `node -v`: v8.12.0
 - `npm -v`: 6.4.1
 - `depcheck --version`: 0.8.4

### Extra info
Currently  creating a PR for that.

**Comment by rumpl** [comment]
Closed with #449
